packages/terra-form-checkbox/docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra Form Checkbox
+
+The Terra Form Checkbox is a responsive input component rendered as a box. When activated, a check mark shall appear. Focus can be activated through tabbing and the checked state can be toggled with the space bar.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-checkbox`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Checkbox from 'terra-form-checkbox';
+
+<Checkbox {id="checkbox" labelText="Default Checkbox" } />
+```
+
+## Component Features
+ * [Cross-Browser Support](https://github.com/cerner/terra-core/wiki/Component-Features#cross-browser-support)
+ * [Responsive Support](https://github.com/cerner/terra-core/wiki/Component-Features#responsive-support)
+ * [Mobile Support](https://github.com/cerner/terra-core/wiki/Component-Features#mobile-support)
+ * [Internationalization Support](https://github.com/cerner/terra-core/wiki/Component-Features#internationalization-i18n-support)
```

Comment:
I don't think this supports Internationalization or Localization

Reply:
True, we can drop those bullet points from here.